### PR TITLE
Bk/update

### PR DESCRIPTION
**Pull request by bkeener7**
## Why is this PR being made?
- Refactor factories to avoid randomly failing tests. DRY up code in test files. Various refactoring/reorganizations/rubocoping, no functional changes other than CircleCI caching.

### CODE REVIEW

File: spec/graphql/mutations/create_user_spec.rb [hunk 1]
```diff
@@ -2,69 +2,56 @@
 
 RSpec.describe Mutations::CreateUser, type: :request do
   describe 'create a user' do
-    describe 'happy path' do
-      it 'can create a user from a mutation' do
-        post '/graphql', params: { query: mutation }
+    let(:parsed_response) { JSON.parse(response.body, symbolize_names: true) }
```
Comment (MarchandMD) [inline spec/graphql/mutations/create_user_spec.rb:5]: adds a reusable `:parsed_response`

File: spec/graphql/mutations/create_user_spec.rb [hunk 2]
```diff
@@ -2,69 +2,56 @@
...
-    createUser(input:{
-      username: ""
-      }) {
-      user {
+  def mutation(username)
+    <<~GQL
+      mutation {
+        createUser(input:{
+          username: "#{username}"
+        }) {
```
Comment (MarchandMD) [inline spec/graphql/mutations/create_user_spec.rb:48]: where does `#{username}` come from?

**Review by MarchandMD — APPROVED** [review]
Looks good; I asked one question; answer if you want. 

I saw a similar behavior in other files, but only need to ask it once...

File: spec/graphql/mutations/create_user_spec.rb (hunk 2 above)
Comment (bkeener7) [reply spec/graphql/mutations/create_user_spec.rb:48]: It's passed through as an argument

```ruby
before { post '/graphql', params: { query: mutation('Floofything') } }
``` 

Or in the case of a blank query, nothing is passed through

```ruby
before { post '/graphql', params: { query: mutation('') } }
```

Comment (bkeener7) [reply spec/graphql/mutations/create_user_spec.rb:48]: In fact, most of the DRY'd up code in this PR was done by implementing arguments into the test queries.

**Pull request merged**